src/operations/map/MapOperations.tsx

```diff
@@ -53,20 +49,43 @@ export class MapOperations {
 
       if (selectedMapType === GlobalInsight.FOOD) {
         const newCountryIso3Data = await countryRepository.getCountryIso3Data(selectedCountryData.properties.iso3);
-        setCountryIso3Data(newCountryIso3Data);
+        if (Array.isArray(newCountryIso3Data) && newCountryIso3Data[1] === 404) {
+          setIsDataAvailable(false);
```

Comment:
This is wrong. Even if we don't have iso3 data but have adm1data we have data and shoudln't zoom out